### PR TITLE
Deflate the "valid" sticker economy and ward off permahungry sec

### PR DESCRIPTION
**Pull request by Equivocateur**
## About the PR
Fixes rigged gloves description to match the blue boxing gloves and not captain gloves.
Renames *most* non-harmful syndicate items to feature less of the word "syndicate" on them to ward off security officers who just KNOW that this particular bag is le bad and makes you valid for perma.
Adds (mostly) non-harmful syndicate items as a **very** rare drop from maintenance lockers.

As for the mosin - its an absurdly low chance to get from a maint locker, and even then it's a meme gun:
image
https://discord.com/channels/310555209753690112/560845886263918612/1126916092762267729

**Media**

Only two syndie entities had spawned in a sea of maint lockers, both from the top part of the pool (operative jumpskirt in the top right and the empty *suspicious* duffel bag in the bottom right):
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
no cl

### CODE REVIEW

File: Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml [hunk 1]
```diff
@@ -218,7 +218,7 @@
   parent: ClothingBackpackDuffelSyndicateBundle
   id: ClothingBackpackDuffelSyndicateEVABundle
   name: syndicate EVA bundle
-  description: "Contains the Syndicate approved EVA suit."
+  description: "Contains the Syndicate approved red EVA suit."
```
Comment (lzk228) [inline Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml:221]: syndi

File: Resources/Prototypes/Entities/Clothing/Hands/gloves.yml [hunk 2]
```diff
@@ -81,7 +81,7 @@
   id: ClothingHandsGlovesBoxingRigged
   name: blue boxing gloves
   suffix: Rigged
-  description: Regal blue gloves, with a nice gold trim. Swanky.
+  description: Blue gloves for competitive boxing.
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Clothing/Hands/gloves.yml:84]: you can just remove name and desc at all, it will be from parent

**Comment by EmoGarbage404** [comment]
Most of the renaming is lame. Just let syndicate items be syndicate items. No need to barely masquerade it when everyone knows what it is.

**Comment by Emisse** [comment]
sounds good to me

**Comment by Equivocateur** [comment]
> Most of the renaming is lame. Just let syndicate items be syndicate items. No need to barely masquerade it when everyone knows what it is.

"It says 'syndicate' here on tin/description" is completely meta imo. The amount of perma/execution-thirsty secoffs eager to jump at the slightest hint of someone being a tot further compounds this. While I do not expect mrp behav in regards to contraband the amount of times security has put someone into perma over a suit or bag is hysterical.

**Comment by Equivocateur** [comment]
image
eternally trolled by chemistry

**Comment by Equivocateur** [comment]
tests failing due to something in chemistry, disregard

**Comment by 0x6273** [comment]
This should have a cl

**Comment by Equivocateur** [comment]
> This should have a cl

Would spark EVERYONE and IMMEDIATELY rush all the maint lockers on the map the moment this pr gets published. It stays a secret simply because of the low odds of syndie loot spawning.

Name/desc changes are too insignificant to warrant a cl entry either.

**Comment by EmoGarbage404** [comment]
> "It says 'syndicate' here on the tin/description" is completely meta imo. The amount of perma/execution-thirsty secoffs eager to jump at the slightest hint of someone being a tot further compounds this. While I do not expect mrp behav in regards to contraband - the amount of times security has put someone into perma over a suit/glasses/moustache/hat or bag is hysterical.

the only reason people consider it meta is because the only ways of obtaining it are confirmations of being a syndicate. You can see how this falls apart the minute you just map some of this stuff in normally. If people are really considering a single item worthy enough of being permabrig/execution then that's just an administration issue.

**Comment by EmoGarbage404** [comment]
> Would spark EVERYONE and IMMEDIATELY rush all the maint lockers on the map the moment this pr gets published. It stays a secret simply because of the low odds of syndie loot spawning.

playing the info game on an open source repo is dumb af. people are just gonna post in #general

**Comment by Equivocateur** [comment]
> > Would spark EVERYONE and IMMEDIATELY rush all the maint lockers on the map the moment this pr gets published. It stays a secret simply because of the low odds of syndie loot spawning.
> 
> playing the info game on an open source repo is dumb af. people are just gonna post in #general

Tacticool suits in the clothesmate still are only mentioned by me or in a conversation started by me on discord so there's that. Eventually people would know that this exists I do not deny, but I'd rather keep it as a surprise.

**Comment by EmoGarbage404** [comment]
it's still lame to do a large rebranding of a bunch of syndicate items, pretend like it's going to be secret for anything more than 2 seconds, then claim that by renaming it "suspicious" people are going to treat it even a shred differently. You're just diluting the aesthetics of what are obvious syndicate items to dupe maybe a handful of new players who have never seen them before.

**Comment by Emisse** [comment]
I have no opinion on the renaming of non stealth items i dont think its gonna change anything, i think the maint locker change is good

**Comment by deltanedas** [comment]
> the only ways of obtaining it are confirmations of being a syndicate

sorry grandpa but syndicate duffelbag is mapped on at least origin, better luck next time

**Comment by Equivocateur** [comment]
> it's still lame to do a large rebranding of a bunch of syndicate items, pretend like it's going to be secret for anything more than 2 seconds, then claim that by renaming it "suspicious" people are going to treat it even a shred differently. You're just diluting the aesthetics of what are obvious syndicate items to dupe maybe a handful of new players who have never seen them before.

I believe I've only rebranded the bag to that - everything else does not have neither that nor anything close to it. Tacticool clothes are basically renamed operative clothes with a miniscule edit and those are treated differently as opposed to actual operative clothing by security. Keeping the word "syndicate" on the tin of items I've changed gives ground to sec chasing you for the entire round just because you wear the wrong clothing despite them not being supposed to know those clothes are syndicate to begin with. 

The entire purpose of the renames is to now have people be valid, and as a result - killed, permad or both for wearing stupid shit. Just because they know these items are syndicate OOC doesnt mean they have vested IC knowledge that this particular black and red bag or that particularly red EVA suit belongs to the syndicate and makes you a confirmed traitor and therefore subject to getting sent to perma.

An esword, a hardsuit, a gun or an eshield as few examples should make you valid, not a particularly black toolbox or bag.

**Comment by deltanedas** [comment]
sus box is still valid bro its robust... robust toolbox

**Comment by Slava0135** [comment]
wait did someone remove space lube from lockers

**Comment by moonheart08** [comment]
> > Would spark EVERYONE and IMMEDIATELY rush all the maint lockers on the map the moment this pr gets published. It stays a secret simply because of the low odds of syndie loot spawning.
> 
> playing the info game on an open source repo is dumb af. people are just gonna post in #general

yes and no
most of the playerbase doesn't *read* #general. It's still pretty effective.

**Comment by Equivocateur** [comment]
> For non-stealth items, there's no reason to not clearly mark them as syndicate items. 
I think equating primarily vanity items in the same group as an esword, python or the blood-red hardsuit to be overkill. 

Having the word "syndicate" slapped on a something that shouldnt have it on it (EVA suit, duffle bag) gives an excuse for security to assume a person is entirely valid while their clothing or item is of the wrong color, thats it. On the other hand things like nocturine bottles, web vests and (d)eswords are obviously contraband and dangerous to others and therefore should be responded to with force. Getting hunted relentlessly by security over the turtleneck, hat or glasses youre wearing because its "not on the list of NT-approved clothing suppliers" sounds completely absurd.

The only exception to my vanity argument is the balloon, whose entire point is to tell the world you're a syndicate agent and don't need no TC to do your work.

**Comment by Equivocateur** [comment]
> The syndicate EVA suit is an EVA suit issued by the syndicate, there's no reason for anyone on an NT station to be causally using one. 

It functions the same as an EVA suit to my knowledge and the only IC different between itself and the ordinary EVA suit is the color and a note on the suit that its "not property of a rival corporation, honest!" (description).

> Security shouldn't assume that someone is a member of the syndicate solely because the have a specific item, but having those items spawn naturally addresses that.

That's not the only thing what I am aiming for - while non-native items that are not inherently dangerous to others or the station itself may warrant a search - they should not cause a neuron activation in a security officer's brain to drop whatever they've been doing and dedicate the next 5-10-30 minutes chasing a person to confiscate their boots/webbing/suit.

I've left the "syndicate" name on things that can be only used for bad activities like the shoulder holster as its only use iirc is holstering guns, grenades and ammo within.

**Comment by deltanedas** [comment]
if syndie eva makes you valid salad it shouldnt be mapped on bagel, that just spurs conflict for no reason

**Comment by Equivocateur** [comment]
> if syndie eva makes you valid salad it shouldnt be mapped on bagel, that just spurs conflict for no reason

Theres a helmet on aspid

**Comment by Equivocateur** [comment]
> It's not about the function, it's about what it means about the wearer. Surely Syndicate stations wouldn't just ignore the presence of someone in a standard issue NT hardsuit.

The syndicate is the informed minority and incredibly hostile, it would make sense for the bad guys to know what's theirs and what's not but not your average Urist McBaton knowing every single item from the uplink roundstart.

> they're not going to know it doesn't have syndicate in the name until they actually catch the person anyway.

How it currently works is an average secoff seeing clothing that is illegally darker or redder in color than the security uniforms and initiates an AOS call on the basis of that. The word "syndicate" in the name is just an excuse while trying to argue that its literally clothing.

**Comment by Equivocateur** [comment]
Fucking fat finger

**Comment by Emisse** [comment]
> if syndie eva makes you valid salad it shouldnt be mapped on bagel, that just spurs conflict for no reason

ambiguity makes gameplay better

**Comment by Anzarot121** [comment]
Adore. It will be very funny to watch how on MRP servers (normal MRP, not English) cops will get out with contraband.

**Comment by Equivocateur** [comment]
> While we can tell them that they don't IC, we can't stop players from knowing OOC what is provided by uplinks. The maint changes of this PR directly address the issues caused by that by creating a non-uplink method of obtaining certain items. 

Let them know OOC, playing dumb IC is part of RP and security does not have the divine knowledge of uplink items roundstart to even say they're contraband.

> Changing the name on the clothing wont have any effects in that scenario other than making it harder for security to explain IC why the person they arrested for willingly and openly wearing contraband

The name changes on some items were made to incite a search rather than spark a manhunt/getting batonned without a word because "it says 'syndicate' here on the item name". 

> If non-stealth items are clearly marked as contraband there is no risk of players getting arrested for wearing something that they could have plausibly believed was just random clothing.

Pointless items should belong in their own category and not be equal to a weapon in the eyes of security players. Getting brigged for a hat as much as for a mosin is ridiculous.

**Comment by Equivocateur** [comment]
EVA, related duffel and generic syndie duffel renamed back to original names. Also addedoutlaw items in the pool, replaced combat gloves with the entire susbox.

**Comment by Equivocateur** [comment]
test fail is unrelated

File: Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml [hunk 3]
```diff
@@ -110,8 +110,8 @@
 - type: entity
   id: CigPackSyndicate
   parent: CigPackBase
-  name: Syndicate Smokes packet
-  description: Elite cigarettes for elite agents. Infused with medicine for when you need to do more than calm your nerves.
+  name: Interdyne herbals packet
+  description: Elite cigarettes for elite people. Infused with medicine for when you need to do more than calm your nerves.
```
[the remark this answers is not in the document]
Comment (Equivocateur) [reply Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml:114]: Nope, a rename of the smokes and thats it.

**Comment by Equivocateur** [comment]
Names/descriptions returned, the cig pack name change is more about flavor than stealth.

File: Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (Equivocateur) [reply Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml]: done

**Comment by retequizzle** [comment]
this should not have been merged with mosins available as maint loot

**Comment by metalgearsloth** [comment]
Roundstart maint raiders gonna be winning even harder.

**Pull request merged**